android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -66,6 +67,14 @@
         views.add(rootLayout);
 
         JSONArray repeatingGroupLayout = jsonObject.getJSONArray(VALUE);
+        int repeatCount = 0;
+        if(jsonObject.has(JsonFormConstants.TEXT)) {
+            repeatCount = jsonObject.getInt(JsonFormConstants.TEXT);
+
+        }
+
+        api = (JsonApi) formFragment.getContext();
```

Comment:
The `JsonApi` can be retrieved from the `WidgetArgs` : `(JsonApi) widgetArgs.getContext()`

Reply:
Fixed.